doc/requirements-doc.txt

```diff
@@ -1,35 +1,34 @@
 # Production requirements. This is what readthedocs.com picks up
 
 git+https://github.com/ray-project/tune-sklearn@master#tune-sklearn
-watchfiles # Required because sphinx-click doesn't support mocking
+watchfiles==0.20.0 # Required because sphinx-click doesn't support mocking
```

Comment:
look like there are some issues with resolving conflicts: https://buildkite.com/ray-project/premerge/builds/8084#018b1ad2-e899-4517-9744-0932f9dacbca/198-364, to update them all to the latest version, I would suggest to unpin all versions and let the ci system resolve the versions itself

The ci system will resolve all conflict and output the content in this job: https://buildkite.com/ray-project/premerge/builds/8084#018b1ad2-e899-4517-9744-0932f9dacbca/198-364, which you can copy the content into this file https://github.com/ray-project/ray/blob/master/python/requirements_compiled.txt

Let me know if these doesn't make sense. Thankks

Reply:
So I just tested unpinning these and constraining them with `requirements_compiled.txt` but that doesn't work; `myst-nb` is constrained by `test-requirements.txt` :/. Same for `myst-parser`. For now I'm not going to spend any time on this until later on when we can come up with a way to reconcile these differences.

Reply:
Ah I see, doc test requirements are declared in test-requirements.txt as well ;). Let me give this a shot and if it works I can put up a PR that you can reuse.

Reply:
My attempt: https://github.com/ray-project/ray/pull/40345